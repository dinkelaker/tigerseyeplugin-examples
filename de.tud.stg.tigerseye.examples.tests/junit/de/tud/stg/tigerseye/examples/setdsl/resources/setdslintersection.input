
Set b = { "2" } â‹‚ { "6", "8", "2" }