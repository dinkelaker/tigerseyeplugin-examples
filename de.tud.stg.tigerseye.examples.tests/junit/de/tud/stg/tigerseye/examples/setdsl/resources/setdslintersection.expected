    Set b = intersection(
asSet(
singleElementedList(
"2")),
asSet(
multiElementedList(
"6",
multiElementedList(
"8",
singleElementedList(
"2")))))